aten/src/ATen/native/cuda/Dropout.cu

```diff
@@ -326,7 +326,9 @@ inline void launcher(
 
 std::tuple<Tensor,Tensor>
 native_dropout_cuda(const Tensor& self, double p, double scale, bool train){
-  TORCH_CHECK(train, "Train parameter is incorrectly set!");
+  if (!train) {
+    return std::tuple<Tensor, Tensor>(self, at::empty(0, self.options().dtype(kBool)));
```

Comment:
For the record, this doesn't look right with returning an empty mask. Backward doesn't take training flag, so we would want mask to be properly set.

But let's not worry much about it here. If it works in your example, it's good enough to be pushed to our branch.
I'm fixing similar issues on upstream dropout PR.